### DISCUSSION TITLE
Rosé Pine Dawn on Obsidian.md

### DISCUSSION THREAD

**Discussion by cutbypham** [rose-pine/rose-pine-theme#103] [Show and tell]
- [Rosé Pine moon](https://github.com/mimishahzad/rose-pine-moon-obsidian)
- Dawn?
**install minimal theme**
![image](https://user-images.githubusercontent.com/40050527/226527050-b98ef688-90a1-4426-b529-9944d69f8e34.png)
**install minimal theme settings plugin** 
![image](https://user-images.githubusercontent.com/40050527/226527165-eab9fbba-130a-4f5a-9d11-f8a3e8c52980.png)
**go to plugin option**
![image](https://user-images.githubusercontent.com/40050527/226527240-10ade0fe-2031-4ddc-bd94-3504c7cea59d.png)

**Comment by sspaeti** [comment]
Just in case, I also created one for Obsidian: https://github.com/sspaeti/obsidian_rose_pine/
